packages/components/select/macro-options.json

```diff
@@ -70,6 +82,20 @@
     "description": "Options for the error message component. The error message component will not display if you use a falsy value for `errorMessage`, for example `false` or `null`.",
     "isComponent": true
   },
+  {
+    "name": "formGroup",
+    "type": "object",
+    "required": false,
+    "description": "Additional options for the form group containing the select component.",
+    "params": [
+      {
+        "name": "classes",
+        "type": "string",
+        "required": false,
+        "description": "Classes to add to the form group (for example to show error state for the whole group)."
```

Comment:
The example in this description, whilst matching the govuk one, doesn't make sense to me, as for the error state you’d use the `errorMessage` option which sets the classes automatically? And what does "whole group" mean here? 🤔